pkg/okteto/k8s.go

```diff
@@ -38,7 +52,7 @@ func getKubernetesTimeout() time.Duration {
 	return timeout
 }
 
-func getK8sClient(clientApiConfig *clientcmdapi.Config) (*kubernetes.Clientset, *rest.Config, error) {
+func GetK8sClientWithApiConfig(clientApiConfig *clientcmdapi.Config) (*kubernetes.Clientset, *rest.Config, error) {
```

Comment:
Is this really needed to be public? I think it can be `getK8sClient` is being called always internally I think